library/src/main/java/com/mvc/imagepicker/ImagePicker.java

```diff
@@ -58,29 +60,71 @@
     private static int minWidthQuality = DEFAULT_MIN_WIDTH_QUALITY;
     private static int minHeightQuality = DEFAULT_MIN_HEIGHT_QUALITY;
 
+    private static String mChooserTitle;
+    private static int mPickImageRequestCode = DEFAULT_REQUEST_CODE;
+    private static boolean mGalleryOnly = false;
```

Comment:
I use your lib in the app but also I need to choose image from gallery on the 'camera screen' so I don't need camera apps but gallery only.